### ISSUE TITLE
Correct Semantic Markup for Section Headings and Tables

### ISSUE THREAD

**Issue by SamiyahKey** [cisagov/manage.get.gov#3179]
### Story

As a user relying on assistive technology,
I need section titles like "Audit Log" and "Registrar" to be properly marked up as headings,
So that I can easily understand and navigate the page's structure and hierarchy.

### Acceptance Criteria

- [ ] Refactor the HTML to move "Audit Log" and "Registrar" outside the < table > tag and mark them as < h2 >.
- [ ] Review other pages with similar issues to ensure consistent markup.
- [ ] Include captions for tables as an optional enhancement to improve usability and context for all users.

**Comment by SamiyahKey** [comment]
@Katherine-Osos just updated the ticket with that screenshot. Also the captions are not 100% part of the remediation but is best practice for adding additional context to the page. I'll separate out into a separate ticket!

**Comment by SamiyahKey** [comment]
@zandercymatics per our slack convo, yes lets do caption and I'll check it out when it's ready! ty

**Comment by witha-c** [comment]
Removed "skipped design" to reflect that an update to design documentation or libraries is not needed.
